### PR TITLE
Rxjs interop updates

### PR DESCRIPTION
**Pull request by atscott**
See individual commits

### CODE REVIEW

**Comment by JeanMeche** [comment]
Are these changes too late for the RC ?

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: fw-core, public-api

**Comment by atscott** [comment]
> Are these changes too late for the RC ?

These kinds of changes are exactly the type of thing we _should_ change in the RC period (also RC isn't until tomorrow). Based on feedback during this period, we can and should update features that will go into the final release.

File: packages/core/rxjs-interop/src/from_observable.ts [hunk 1]
```diff
@@ -53,20 +56,23 @@ export function fromObservable<T>(source: Observable<T>): Signal<T>;
...
 export function fromObservable<T, U extends T|null|undefined>(
     // fromObservable(Observable<Animal>) -> Signal<Cat>
-    source: Observable<T>, initialValue: U): Signal<T|U>;
-export function fromObservable<T, U = never>(source: Observable<T>, initialValue?: U): Signal<T|U> {
+    source: Observable<T>, options: {initialValue: U, requireSync?: false}): Signal<T|U>;
+export function fromObservable<T>(
+    // fromObservable(Observable<Animal>) -> Signal<Cat>
+    source: Observable<T>, options: {requireSync: true}): Signal<T>;
+export function fromObservable<T, U = undefined>(
+    source: Observable<T>, options?: {initialValue?: U, requireSync?: boolean}): Signal<T|U> {
```
Comment (dylhunn) [inline packages/core/rxjs-interop/src/from_observable.ts:64]: Nit: I'd add an explanatory comment that this signature has to be listed second

**Review by dylhunn — APPROVED** [review]
reviewed-for: fw-core, public-api

File: packages/core/rxjs-interop/src/from_observable.ts (hunk 1 above)
Comment (atscott) [reply packages/core/rxjs-interop/src/from_observable.ts:65]: It doesn't though, does it? The first two can be in either order. Or do you mean the last one has to be listed third?

File: packages/core/rxjs-interop/src/to_observable.ts [hunk 2]
```diff
@@ -10,11 +10,11 @@ import {assertInInjectionContext, effect, inject, Injector, Signal} from '@angul
 import {Observable} from 'rxjs';
 
 /**
- * Options for `fromSignal`.
+ * Options for `toObservable`.
  *
  * @developerPreview
  */
-export interface FromSignalOptions {
+export interface toObservableOptions {
```
Comment (MaximSagan) [inline packages/core/rxjs-interop/src/to_observable.ts:17]: Pascal

**Comment by AndrewKushnir** [comment]
**Caretaker note**: adding the "merge" label based on the conversation with @atscott.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 1dddb7878688e413ddbc7c1fd767bea44d675b69.

**Pull request closed without merging**